processing/src/main/java/org/apache/druid/query/expression/LookupExprMacro.java

```diff
@@ -36,6 +37,7 @@
 
 public class LookupExprMacro implements ExprMacroTable.ExprMacro
 {
+  private static final byte LOOKUP_EXPR_KEY = 0x01;
```

Comment:
Can we centralize these? Otherwise I worry about accidentally reusing IDs.

Reply:
I was worried about that too, but hesitated since `Expr` is in core and this is in processing and also some others such as the `DimensionSpec` have this problem, but yeah I agree and fixed it and moved both of them to `Exprs` because wasn't sure where else to put it and it seemed sort of funny on `Expr`.